simtools/corsika/corsika_output.py

```diff
@@ -874,13 +967,13 @@ def num_photons_per_event(self):
         numpy.array
             Number of photons per event.
         """
-        if self._num_photons_per_event is None:
-            self._num_photons_per_event = np.sum(self.num_photons_per_event_per_telescope, axis=1)
+        self._num_photons_per_event = np.sum(self.num_photons_per_event_per_telescope, axis=0)
         return self._num_photons_per_event
 
     def get_num_photons_distr(self, bins=50, range=None, event_or_telescope="event"):
         """
-        Get the distribution of photons per event.
+        Get the distribution of photons per event or per telescope, depending on the string
```

Comment:
I actually consider the numpy function confusing, since we always think in `X and Y` when plotting and not in `Y and X`. Since it is well documented here and the function does not have the same name as the numpy one, I would prefer to keep this order, unless you have a strong opinion

Reply:
I don't have a strong opinion but I am not sure why you mention `x` and `y` when in this case the point is about the histogram values and its edges (in which case the order between the two is fairly arbitrary).

Reply:
I now inverted the order in the files. indeed, it is rather arbitrary and not a convention. Note that `ax.pcolormesh` is written as ` ax.pcolormesh(x_edges[i_hist], y_edges[i_hist], hist_values[i_hist]` and `ax.bar` also `        ax.bar(edges[i_hist][:-1], hist_values[i_hist],`

Reply:
Right, when creating a figure it's reversed (which also makes sense), but when binning the data it's `hist, edges`. So now look how nice and consistent it looks (while earlier it was reversed):
```
if event_or_telescope == "event":
   hist, edges = np.histogram(self.num_photons_per_event, bins=bins, range=range)
return hist, edges
```